package.json

```diff
@@ -35,7 +35,7 @@
     "@jupiterone/eslint-config": "^2.1.0",
     "@philidem/rewrite-relative-paths": "^1.2.3",
     "babel-jest": "^29.3.1",
-    "babel-plugin-module-resolver": "^4.1.0",
+    "babel-plugin-module-resolver": "^5.0.0",
```

Comment:
https://github.com/tleunen/babel-plugin-module-resolver/blob/master/CHANGELOG.md#500-2023-01-09

This drops support for node < 16